accessibility-101.md

```diff
@@ -0,0 +1,196 @@
+ This would be true for blind visitors to your website, but also when your image can't be loaded for some reason.
+ Not adding an alt-tag to an img-attribute is therefore not only breaking accessibility, but going against the HTML5-spec. 
+ 
+ I implore any web developer who catches themselves doing this to eat their programmer's hat and work on Windows 95 exclusively for a week. After the time is up, write an essay on what you have learnt from this ordeal so I can have a laugh during my afternoon coffee. :)
+
+ Now, there is one caveat here. Alt-attributes are mandatory according to the HTML5-spec, but it's not mandatory to actually fill them in. `<img src="awesome-image.jpg", alt="">` is therefore legal HTML5 code.
+ 
+ Should you therefore fill in alt-tags for all images? It depends on the image, really. For background images, the answer is usually no, but you should use CSS for those anyway.
+ 
+ For purely decorative images that have no information in them at all, you're basically free to choose. Either put in something useful and descriptive or nothing at all.
+ 
+ For images that contain information, like a brochure, a map, a chart etc., not adding alt text breaks WCAG unless you provide a textual alternative. This is usually an alt-attribute, but can also be a block of text right below or next to the image.
+ 
+ For images of text, the text can either be included in the alt-attribute or offered in some alternative manner. The problem is that adding the textual alternative on the same page would basically make the same content show twice for people who can see the image, which is why the alt-attribute is better in this case.
+ 
+ 
+####I can't read your scrawl, son
+ 
+ Even people who don't wear glasses and have no problem with their eyesight at all benefit from an easy to read font and proper contrast. 
+ I'm sure you would cringe if you had to fill in a form where light yellow, hopelessly loopy letters are placed on a white background. For people who's eyesight is not as good, like your grandma for example, this becomes hopelessly worse. 
+
+ The WCAG has contrast ratios for smaller and larger letters and there's plenty of tools out there to check if the contrast ratios are strong enough. The information and tooling is there, go use it :)
+ 
+####What does this button do?
+ While we are at the topic of forms, let's quickly glance at the <label> tag. This little guy is kinda important.
+ When you put some input fields on a web page, you can use labels to ...well ...label them. However, putting them next to each other is not quite enough. The attribute you want is the for-attribute, which takes the ID of a subsequent input field. This way, assistive technologies know what label to associate with what form field.
+ I guess the best way to illustrate this is by giving an example:
+ 
+```
```

Comment:
Yes for the wiki you use master, for the FCC you use staging.

for the code blocks you can specify what type of code it is like css, html, javascript or js and so on.

As for the heading it might just be me then, My linter adds space to them and so I have gotten used to that as the standard, it also looks better when reading the markdown code.

However, notice the back ticks, there was a whole section with headings and such that was rendered as a quotation or something else. Could be because of that. I used the live preview here by the way.